.github/config.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  👋 Welcome, and thank you for your first contribution!
+
+  We ask you to take a moment to read through our Contributing Guide:
+
+  👉 https://github.com/nipreps/{repository}/blob/main/CONTRIBUTING.md
```

Comment:
I've seen that there is a [cookiecutter repository](https://github.com/nipreps/Cookiecutter), but its contents seem not to be ready to self-configure an actual repository. So I did not figure out a better way to mean "your NiPreps repository" here.